AdblockPlusSafariExtension/ActionRequestHandler.m

```diff
@@ -29,6 +29,14 @@ - (void)beginRequestWithExtensionContext:(NSExtensionContext *)context
 {
   AdblockPlus *adblockPlus = [[AdblockPlus alloc] init];
   adblockPlus.activated = YES;
+  adblockPlus.lastActivity = [[NSDate alloc] init];
+
+  if (adblockPlus.performingActivityTest) {
+    // Cancel the reloading. This will reduce time of execution of execution of content blocker
+    // and therefore host application will be notified about result of testing ASAP.
+    [context cancelRequestWithError:[NSError errorWithDomain:AdblockPlusErrorDomain code:AdblockPlusErrorCodeActivityTest userInfo:nil]];
```

Comment:
This should really be hidden in a separate compilation unit, with all other activity test related hack code.